cmd/gaia/init/init.go

```diff
@@ -46,11 +45,11 @@ func displayInfo(cdc *codec.Codec, info printInfo) error {
 // nolint
 func InitCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "init",
+		Use:   "init <moniker>",
```

Comment:
Don't we use `[...]` ?

Reply:
`grep -rw Use:` would show how inconsistent our approach has been so far. I'm ok with `[moniker]` though

Reply:
addressed

Reply:
We need to unify the approach here.

Reply:
yeah idc really about either, just as long as we're consistent.